query/api/src/main/java/org/codice/ddf/admin/api/fields/FunctionField.java

```diff
@@ -38,15 +36,9 @@
 
   T getReturnType();
 
-  void setArguments(Map<String, Object> value);
-
-  FunctionReport<T> execute();
+  FunctionReport<T> execute(Map<String, Object> value, List<Object> path);
```

Comment:
❓ Why are we passing in the path here as an argument? Seems a little odd.

Reply:
Similar to `setArgument`, I didn't see the benefit of keeping it as a separate method. No one was leveraging it and the path should always be set before
 execute is called.